pkg/services/hivelocity/client/hvclient.go

```diff
@@ -149,7 +134,11 @@ func (c *realClient) DeleteDevice(ctx context.Context, deviceID int32) error {
 }
 
 func (c *realClient) ShutdownDevice(ctx context.Context, deviceID int32) error {
-	return fmt.Errorf("todo ShutdownDevice")
+	_, _, err := c.client.DeviceApi.PostPowerResource(ctx, deviceID, "shutdown", nil) //nolint:bodyclose // Close() gets done in client
+	if err != nil {
+		return fmt.Errorf("[ProvisionDevice] PostPowerResource() failed. deviceID %d: %w", deviceID, err)
```

Comment:
Typo: should be `[ShutdownDevice]`. Maybe we should think about this way of creating log messages. The text in the braces is redudant. And redundancy will most break sooner or later...